Core/Core/Extensions/UICollectionViewExtensions.swift

```diff
@@ -37,7 +37,7 @@ extension UICollectionView {
 
     /// Registers a reusable cell object.
     /// This assumes that the reuse identifier matches the type name.
-    func registerCell(_ cellClass: AnyClass, bundle: Bundle? = nil) {
+    public func registerCell(_ cellClass: AnyClass, bundle: Bundle? = nil) {
```

Comment:
I don't know why this needs a bundle passed in. Won't the bundle always be the one `cellClass` is defined in?